automation/env-build/aws-security.tf

```diff
@@ -43,6 +43,20 @@ resource "aws_security_group" "transparent-security-img-sg" {
     protocol = "tcp"
   }
 
+  ingress {
```

Comment:
Do these ports really need to be opened for building the environment?

Reply:
Maybe not. Let me test with new AMI and see if isn't causing any issues.I will be re-adding setup_hcp.yml in my next pull.